runtimes/pythonrt/py.go

```diff
@@ -331,6 +333,8 @@ func pyExports(ctx context.Context, pyExe string, fsys fs.FS) ([]Export, error)
 		return nil, fmt.Errorf("inspect: %w.\nPython output: %s", err, buf.String())
 	}
 
+	sdklogger.Warn(ctx, "inspect output", zap.String("output", buf.String()))
```

Comment:
I prefer to pass the logger from pythonrt

Reply:
no idea why i even added this warn